build.sbt

```diff
@@ -77,7 +78,8 @@ lazy val derivationBase = crossModule("derivation", CrossType.Full)
       "io.circe" %%% "circe-testing" % circeVersion % Test
     ),
     ghpagesNoJekyll := true,
-    docMappingsApiDir := "api"
+    docMappingsApiDir := "api",
+    addCompilerPlugin("org.scalamacros" % "paradise" % paradiseVersion cross CrossVersion.patch)
```

Comment:
Hmm, ugh, I forgot about the plugin. I think we should go ahead and merge this, but we might consider splitting it out into a separate module before 1.0.

If I remember correctly, though, there should be no problem depending on and using circe-derivation without enabling the plugin as long as you don't use the annotations? I should definitely confirm this.

Reply:
Yes, very much like `circe-generic` the plugin should only be needed if you use of the annotation. That should be documented though.